### ISSUE TITLE
constructor scope and getter/setter scope overlaps

### ISSUE THREAD

**Issue by Fee0** [rusty-ecma/RESSA#85]
It seems that the scope of constructors somehow overlap with the scope inside getter and setter (setter parameter seem to work).

```
class e {
    constructor(t) {}

    get a() {
        let t;
    }

    get b(x) {
        let t;
    }
}
```

**Comment by FreeMasen** [comment]
Thank you for the bug report, I will try and figure out what is happening here this weekend.

**Comment by FreeMasen** [comment]
This was merged as part of `0.8.1`
